demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -595,11 +596,9 @@ class ContentList extends React.Component {
                       (isSavingOrUpdating ? "Updating..." : "Update content") :
                       (isSavingOrUpdating ? "Saving..." : "Store content")}
                   />,
-                  <Button
-                    className={"choice-input-button content-button link-button" +
-                      (displayGeneratedLink ? " enabled" : "")}
-                    onClick={onClickGenerateLink}
-                    value={0xf0c1}
+                  <GeneratedLink
+                    enabledGenerateLink={displayGeneratedLink}
+                    onClickGenerateLink={onClickGenerateLink}
```

Comment:
Why not just `enabled` and `onClick`? This repetition of the role of the button is unneeded.